### DISCUSSION TITLE
I want to create a simple config for my server-side mod. Is there a library mod that anyone would recommend?

### DISCUSSION THREAD

**Discussion by TechPro424** [FabricMC/fabric-api#2841] [Mod Dev Support]
I want to create a simple config for my server-side mod
It basically needs to be an array with a list of strings, and I will check if the array includes certain strings for my mod's functionality

Is there a library mod that anyone would recommend, or do I just create a json file and read it manually?

Please ping me when replying

**Comment by KosmX** [comment]
There are multiple config libraries, mostly annotation based:  
https://fabricmc.net/wiki/tutorial:libraries
